GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripWINONLY.py

```diff
@@ -524,13 +549,195 @@ def calladdusrtosys():
+                    def completedPOP(username):
+                        COMPLETE = QMessageBox()
+                        COMPLETE.setIcon(QMessageBox.Question)
+                        COMPLETE.setWindowTitle('Hey! Listen!')
+                        COMPLETE.setText(
+                            'User ' + username + ' has been successfully removed from the system.')
+                        COMPLETE.setStandardButtons(QMessageBox.Close)
+                        COMPLETE.exec_()
+                        self.close()
+
+                    if len(username) == 0 or username not in listoNames:
+                        if len(username) == 0:
+                            print('no username entered')
+                            ERROR_NO_USER = QMessageBox()
+                            ERROR_NO_USER.setIcon(QMessageBox.Warning)
+                            ERROR_NO_USER.setWindowTitle('Hey! Listen!')
+                            ERROR_NO_USER.setText(
+                                'No username was entered. No user was removed.')
+                            ERROR_NO_USER.setStandardButtons(QMessageBox.Close)
+                            ERROR_NO_USER.exec_()
+                        else:
+                            print('User not found')
+                            ERROR_NO_USER_FOUND = QMessageBox()
+                            ERROR_NO_USER_FOUND.setIcon(QMessageBox.Warning)
+                            ERROR_NO_USER_FOUND.setWindowTitle('Hey! Listen!')
+                            ERROR_NO_USER_FOUND.setText(
+                                'User was not found on system. No user was removed.')
+                            ERROR_NO_USER_FOUND.setStandardButtons(QMessageBox.Close)
+                            ERROR_NO_USER_FOUND.exec_()
+                    else:
+                        print('User was removed............ This is a test')
+                        # Executes command.
+                        sub.Popen(["powershell", "& {net user " + username + " /DELETE}"])
+                        completedPOP(username)
+
+                def confirmation():
+                    CONFIRM = QMessageBox()
+                    CONFIRM.setWindowTitle('Hey! Listen!')
+                    CONFIRM.setText("Hey! Are you sure you want to do this?")
+                    CONFIRM.setIcon(QMessageBox.Question)
+                    CONFIRM.setStandardButtons(QMessageBox.No | QMessageBox.Yes)
+                    CONFIRM.setWindowIcon(QtGui.QIcon(':/Pictures/images/HEY.png'))
+                    x = CONFIRM.exec_()
+                    if x == QMessageBox.Yes:
+                        print('Removing user...')
+                        removal()
+                    elif x == QMessageBox.No:
+                        print('Cancelling...')
+
+                def cancel_button():
+                    self.close()
+
+                self.Cancel_button.clicked.connect(cancel_button)
+                self.Confirm_button.clicked.connect(lambda: confirmation())
```

Comment:
Codacy found an issue: [Lambda may not be necessary](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5927419)